src/components/OrderBook/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useRef } from 'react'
+import { OrderBookStyled as Wrapper, Header, Book, Orders, Order, Spread } from './OrderBook.styled'
+import { dummyOrders, dummyPrice } from 'components/OrderBook/dummyTradingData'
+interface OrdersType {
+  readonly orders?: typeof dummyOrders
+}
+
+export const OrderBook: React.FC<OrdersType> = ({ orders }) => {
+  const SpreadEl = useRef<HTMLInputElement>(null)
+  useEffect(() => {
+    SpreadEl.current &&
+      SpreadEl.current.scrollIntoView({
+        behavior: 'smooth',
+        block: 'center',
+        inline: 'center',
+      })
+  }, [])
+
+  return (
+    <Wrapper>
+      <Book>
+        <Header className="header">
+          <span>Price(USDT)</span>
+          <span>Amount(WETH)</span>
+          <span>Total</span>
+        </Header>
+
+        <Orders className="sell">
```

Comment:
do we want the parent container if `!orders`?

Reply:
Yes and as content show 'no orders yet'. This way the user knows what to expect there in terms of functionality.